contrib/cirrus/release_cache.sh

```diff
@@ -0,0 +1 @@
+contrib/cirrus/cache_release.sh
```

Comment:
This can't possibly work; the symlink should not include `contrib/cirrus`

Reply:
Woops :blush:

Reply:
got it